README.md

```diff
@@ -57,23 +53,35 @@ If you use any of these files, please carefully follow the [instructions for cit
 To prepare the environment to run the PhenoPLIER code, follow the steps in [environment](environment/).
 This will create a conda environment and download the necessary data.
 Depending on your Internet speed, this shouldn't take more than 24 hours.
+We tested the code in Ubuntu 20.04+.
 
-We tested the code in Ubuntu 18.04+.
-Alternatively, you can use our Docker image (see below), which will greatly simplify running the code.
+**We strongly recommend** using our Docker image (see below), which will greatly simplify running the code and make sure you use the same environment for the analyses.
 
 ### Hardware requirements
 
-The code was run with an Intel Core i5 (4 cores) and 64 GB of RAM (32 GB should be enough also).
-The setup will download ~70 GB of input data and generate ~45 GB of results files, so you would need to have at least 120 GB if you plan to run all the steps.
-Running all the steps would take around a week under this hardware configuration.
+Most of the code was run with an Intel Core i5 (4 cores) and 64 GB of RAM (32 GB should be enough also).
+The null simulations and gene-gene correlation matrices for the GLS model were computed using a cluster.
```

Comment:
Maybe add a couple of details about the cluster?

Reply:
Yep, added, thanks!